### DISCUSSION TITLE
How to optmize gifs?

### DISCUSSION THREAD

**Discussion by lhguerra** [cloudinary-community/next-cloudinary#318] [Q&A]
When dealing directly with a Cloudinary url, I use this option to optimize gifs: `fl_lossy`
How can I pass it to CldImage component?

**Comment by lhguerra** [comment]
Well, while submiting this question it came to me that lossy is a flag, so I just added `flags={['lossy']}` to the component and it worked! :tada:

**Reply by colbyfayock** [reply]
Haha perfect!
